paystell-frontend/src/components/transaction/TransactionSummary.tsx

```
@@ -0,0 +1,383 @@
+}: TransactionSummaryProps) {
+  const [feeEstimation, setFeeEstimation] = useState<FeeEstimation | null>(null);
+  const [copied, setCopied] = useState(false);
+
+  const {
+    sourceAddress,
+    destinationAddress,
+    amount,
+    currency = 'XLM',
+    memo,
+    fee,
+    network = 'testnet',
```

Biome (2.1.2)

[error] Shouldn't redeclare 'network'. Consider to delete it or rename it.

'network' is defined here:


(lint/suspicious/noRedeclare)